config/manager/kustomization.yaml

```diff
@@ -12,5 +12,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: quay.io/opdev/operator-certification-operator
-  newTag: 0.0.0
+  newName: quay.io/acornett/operator-certification-operator
```

Comment:
Don't think we want to change this?

Reply:
Correct, I thought this was shelved, but it was checked in. Will revert.